api/src/controllers/referent.js

```diff
@@ -864,4 +865,78 @@ router.get("/:id/session-phase1", passport.authenticate("referent", { session: f
   }
 });
 
+router.put("/young/:id/phase1Status/:document", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const keys = ["cohesionStayMedical", "autoTestPCR", "imageRight", "rules"];
+    const { error: documentError, value: document } = Joi.string()
+      .required()
+      .valid(...keys)
+      .validate(req.params.document);
+    if (documentError) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS });
+
+    const young = await YoungModel.findById(req.params.id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.NOT_FOUND });
+
+    let values;
+    if (document !== "cohesionStayMedical") {
+      const { error: bodyError, value } = Joi.object({
+        [`${document}FilesStatus`]: Joi.string()
+          .trim()
+          .valid(FILE_STATUS_PHASE1.TO_UPLOAD, FILE_STATUS_PHASE1.WAITING_VERIFICATION, FILE_STATUS_PHASE1.WAITING_CORRECTION, FILE_STATUS_PHASE1.VALIDATED)
+          .required(),
+        [`${document}FilesComment`]: Joi.alternatives().conditional(`${document}FilesStatus`, {
+          is: FILE_STATUS_PHASE1.WAITING_CORRECTION,
+          then: Joi.string().trim().required(),
+          otherwise: Joi.isError(new Error()),
+        }),
+      }).validate(req.body);
+      if (bodyError) return res.status(400).send({ ok: false, code: bodyError });
+      if (!value[`${document}FilesComment`]) value[`${document}FilesComment`] = undefined;
+      values = value;
+    } else {
```

Comment:
et ce serait un `else if` ici ducoup :
```suggestion
    } else if(document === "cohesionStayMedical") {
```